src/main/java/com/parasoft/findings/jenkins/parser/FlowAnalysisPathBuilder.java

```diff
@@ -292,18 +285,6 @@ private static void addStyledMessage(StringBuilder sb, String message, boolean b
         }
     }
 
-    private void addTrackedMessage(StringBuilder sb, IFlowAnalysisPathElement descriptor, String attribute, boolean bFullDescription)
-    {
-        String message = _violation.getTrackedVariablesMessages().get(attribute);
-        String variables = descriptor.getProperties().get(attribute);
-
-        if (StringUtil.isEmpty(variables)) {
-            return;
-        }
-        String trackedMessage = message + IStringConstants.COLON_SP + variables;
```

Comment:
`IStringConstants.COLON_SP` in parasoft-findings-utils is unused now and can be removed.